internal/pkg/externalplugins/tars/tars.go

```diff
@@ -266,18 +254,46 @@ func HandleAll(log *logrus.Entry, ghc githubClient, config *plugins.Configuratio
 		log.Warnf("No repos have been configured for the %s plugin", PluginName)
 		return nil
 	}
-	var buf bytes.Buffer
-	fmt.Fprint(&buf, "archived:false is:pr is:open sort:created-asc")
+
+	var queries []string
 	for _, org := range orgs {
-		fmt.Fprintf(&buf, " org:\"%s\"", org)
+		queries = append(queries, searchQueryPrefix+` org:"`+org+`"`)
 	}
-	for _, repo := range repos {
-		fmt.Fprintf(&buf, " repo:\"%s\"", repo)
+
+	if len(repos) > 0 {
+		var reposQuery bytes.Buffer
+		fmt.Fprint(&reposQuery, searchQueryPrefix)
+		for _, repo := range repos {
+			slashSplit := strings.Split(repo, "/")
+			if n := len(slashSplit); n != 2 {
+				log.WithField("repo", repo).Warn("Found repo that was not in org/repo format, ignoring...")
+				continue
+			}
+			org := slashSplit[0]
+			repoName := slashSplit[1]
+			tars := externalConfig.TarsFor(org, repoName)
+			fmt.Fprintf(&reposQuery, " label:\"%s\" repo:\"%s\"", tars.OnlyWhenLabel, repo)
```

Comment:
If multiple repos are traversed, the query string becomes `label:"labelA" repo:"repoA" label:"labelA" repo:"repoB"` Does it meet expectations?

Reply:
Fixed.